src/csharp/Grpc.Tools/build/_protobuf/Google.Protobuf.Tools.targets

```diff
@@ -80,7 +80,7 @@
            >$(Protobuf_PackagedToolsPath)/$(Protobuf_ToolsOs)_$(Protobuf_ToolsCpu)/protoc</Protobuf_ProtocFullPath>
     </PropertyGroup>
 
-    <Error Condition=" '$(DesignTimeBuild)' != 'true' and '$(PROTOBUF_PROTOC)' == ''
+    <Error Condition=" '$(DisableProtobufDesignTimeBuild)' != 'true' and '$(PROTOBUF_PROTOC)' == ''
```

Comment:
have we considered having a property that's named in "positive form" (as opposed to `DisableXYZ` that is true by default and can be set to false if the use wants?  Perhaps that's more idiomatic?

Reply:
It looks like majority of common boolean msbuild properties are named in the positive form, but there are a few that are named `Disable...` too. WDYT?
https://docs.microsoft.com/en-us/visualstudio/msbuild/common-msbuild-project-properties?view=vs-2017

Reply:
Both exist and it's up to preference of the developer. I'll defer to your judgement here. I don't think this should be a major issue since we envision design time build to only be turned off in rare scenarios.